tools/fiptool/fiptool.c

```diff
@@ -52,34 +52,6 @@
 #define OPT_PLAT_TOC_FLAGS 1
 #define OPT_ALIGN 2
 
-static image_desc_t *lookup_image_desc_from_uuid(const uuid_t *uuid);
```

Comment:
I prefer global data to be at the top, so one can easily find them/see them when opening the file before reading any of the code.